constants/auth-constants.ts

```diff
@@ -0,0 +1,5 @@
+export enum SessionStatus {
+  Authenticated = 'unauthenticated',
```

Comment:
looks like a typo here

Reply:
for Authenticated it looks like its value shouldn't be `unauthenticated`

Reply:
Oh, I see it now. Thanks!